core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -512,6 +535,9 @@ private[spark] class TaskSetManager(
           serializedTask)
       }
     } else {
+      if (runningTasks >= maxConcurrentTasks) {
+        logDebug("Already running max. no. of concurrent tasks.")
```

Comment:
can you update this msg to include the jobGroup and full config name?

also I think it would be good to have this log at the info level one time (not on every `resourceOffer`, just the first time we hit the limit for this TaskSet).  I just see users hitting this inadvertently and so it would be good to make it clear.

Reply:
I'll make the change for this and also update any comments to explain the behavior so far. Also I am not clear on the earlier reply as to what was the resolution for accounting the activeJobId. Do you still have any inputs ?